Documentation/Books/Manual/Architecture/DeploymentModes/Cluster/Architecture.md

```diff
@@ -1,29 +1,33 @@
 Cluster Architecture
 ====================
 
-The cluster architecture of ArangoDB is a _CP_ master/master model with no
-single point of failure. With "CP" we mean that in the presence of a
+The Cluster architecture of ArangoDB is a _CP_ master/master model with no
+single point of failure. 
+
+With "CP" we mean that in the presence of a
```

Comment:
Explain the origin of this two letter word
```suggestion
With "CP" in terms of the [CAP theorem](https://en.wikipedia.org/wiki/CAP_theorem)
we mean that in the presence of a
```

Reply:
thanks simran